src/rev_ai/streaming.py

```diff
@@ -84,4 +84,4 @@ def received_message(self, m):
     # c = RevClient("ws://localhost:8080/client/ws/speech")
```

Comment:
Consider deleting this file and restoring it later when the service is turned on